modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```
@@ -2232,6 +2232,7 @@ before-the-event-by=before the event by:
 begin-a-workflow=Begin a workflow.
 begin-date=Begin Date
 begin-each-line-with=Begin each line with
+begin-typing-inside-double-quotes-to-see-the-autocompletion-options-or-press-xx=Begin typing inside double quotes to see the autocompletion options, or press xx.
```

Currently there's no hotkey to open autocomplete besides typing in the quotes. So I'd leave out the last part, "Begin typing inside double quotes to see the autocompletion options."

```suggestion
begin-typing-inside-double-quotes-to-see-the-autocompletion-options=Begin typing inside double quotes to see the autocompletion options.
```